src/Downloader.ts

```diff
@@ -586,6 +590,10 @@ class Downloader {
   }
 
   private async getCompressedBody(resp: any): Promise<any> {
+    const content_type = resp.headers['content-type'];
+    if (this.webp && (content_type.includes('jpeg') || content_type.includes('png') || content_type.includes('jpg') || content_type.includes('gif'))) {
+      imageminOptions.plugins.push(imageminWebp({ quality: 75 }))
+    }
     return this.isMimeTypeImage(resp.headers['content-type']) ? await imagemin.buffer(resp.data, imageminOptions) : resp.data;
```

Comment:
This part and `imageminOptions` should be rewritten based on our discussion on Slack. Should be a one line method.